src/assets/icons/calendar.svg

```diff
@@ -1,5 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-  <g fill-rule="evenodd" transform="translate(-224 -192)">
-    <path fill="#95999D" d="M237,193 L240,193 L240,195 L224,195 L224,193 L227,193 L227,192 L229,192 L229,193 L235,193 L235,192 L237,192 L237,193 Z M224,196 L240,196 L240,206.997527 C240,207.551177 239.555369,208 238.999128,208 L225.000872,208 C224.448106,208 224,207.544239 224,206.997527 L224,196 Z M226,198 L226,206 L238,206 L238,198 L226,198 Z M232,201 L237,201 L237,205 L232,205 L232,201 Z"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16">
```

Comment:
Removing `viewBox` attribute is not a good idea. Giving the circumstances, the SVG will not respect its own ratio. We had the issue lately. It may be too restrictive. SVGO doesn't remove it by default usually.

Reply:
I don't know enough of SVG rules to configure the svgomg tool. (for instance it says that `height` and `width` is redondant with `viewBox`. So I will close this PR as I cannot manage the good way to enhance SVG files.